src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -194,4 +208,34 @@ private function getErrorMessages(Exception $e): array
             ),
         ];
     }
+
+    /**
+     * @return bool
+     */
+    private function isProductPageV2Enabled(): bool
+    {
+        $productPageV2FeatureFlag = $this->get('prestashop.core.feature_flags.modifier')
+            ->getOneFeatureFlagByName(\PrestaShopBundle\Controller\Admin\ProductController::FEATURE_FLAG_PRODUCT_PAGE_V2);
```

Comment:
Move to use section

Reply:
Ping @matks and it's ok ;)

Reply:
Too fast for me 😛  look at https://github.com/PrestaShop/PrestaShop/pull/24038/commits/392e403b7904bd000c125010415b18f7b9642b6f